src/JSDOMDomProvider.js

```diff
@@ -61,6 +77,17 @@ export default class JSDOMDomProvider {
     return this.dom.window.happoProcessor.init({ targetName });
```

Comment:
I think inside the happoProcessor (which is src/browser/processor.js) we need to change some things so that the cursor is reset (and possibly other things as well).